file-formats/chkc/type/zif_atc_aff_chkc_v1.intf.abap

```diff
@@ -0,0 +1,21 @@
+INTERFACE zif_atc_aff_chkc_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Object Type CHKC</p>
+    "! CHKC object type
```

Comment:
Small suggestion to replace CHKC by the name of the object type "ATC check category"

```suggestion
    "! <p class="shorttext">ATC Check Category</p>
    "! ATC an check category properties
```

Reply:
Done.

Reply:
Is the 'an' in your description intentional? I removed it in my commit. But I can add it, if it should be there.

Reply:
Thanks, without 'an' is fine